python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -706,3 +714,29 @@ def train_end(self, estimator, *args, **kwargs):
             estimator.logger.info('[Epoch %d] EarlyStoppingHanlder: '
                                   'early stopping due to %s not improving',
                                   self.stopped_epoch, self.monitor.get()[0])
+
+class GradientUpdateHandler(BatchEnd):
+    """Gradient Update Handler that apply gradients on network weights
+
+    :py:class:`GradientUpdateHandler` takes the priority level. It updates weight parameters
+    at the end of each batch
+
+    Parameters
+    ----------
+    priority : scalar, default -np.Inf
```

Comment:
Should a integer be used instead of `-np.Inf`? Maybe there are some use-cases where a handler should be run prior to the gradient handler?